submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,33 @@
+
+/* Toggle menu */
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+hamburgerToggle.addEventListener('click', function (e) {
+  
+  e.preventDefault();
+
+  document.querySelector('.flexmenu').classList.toggle('active');
```

Comment:
Please move this construction to the global constant:
` document.querySelector('.flexmenu')`
No need to query this element on each click. It will be cached when you'll move it to the global scope